src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBody.java

```diff
@@ -43,35 +47,35 @@
     @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate currentDate;
 
-    @JsonProperty("orderDirections1isIncluded")
     private boolean hasFirstOrderDirections;
 
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
     @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate docUploadDeadline;
 
-    @JsonProperty("orderDirections2isIncluded")
     private boolean hasSecondOrderDirections;
 
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
     @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate eyewitnessUploadDeadline;
 
-    @JsonProperty("hearingisRequired")
-    private boolean hearingRequired;
+    @JsonProperty("hearingRequired")
+    private boolean paperDetermination;
```

Comment:
variable name and json property looks a lot different, can we use same for both?

Reply:
not right now, it needs to be updated in the template and that can happen only after the code itself is merged.  basically we need to  update code -> update template -> update code again to clean up

Reply:
Thanks 👍